docs/flashbots-auction/advanced/gas-fee-refunds.md

```diff
@@ -73,27 +73,45 @@ Bundles sent by the same signer will be treated as non-competitive.
 
 ### The Flat Tax Rule
 
-<div className="med caption-img">
-
-![Flat tax rule](/img/flat-tax-rule.png)
-
-Definition of the flat tax rule
-
-</div>
+- **$B(T)$** is the most profitable block produced from bundles in $T$.
+- **$v(T)$** is the value of $B(T)$.
+- **$b_i(T)$** is the payment of all bundles sent by identity $i$ if block $B(T)$ is realized.
+- **$\mu_i(T) = \min\{b_i(T), v(T) - v(T \setminus \{i\})\}$** is the marginal contribution of all bundles sent by identity $i$ if $B(T)$ is realized. We bound the marginal contribution so that the net payment can't be negative.
```

Comment:
this doesn't render this brackets, I would try. \min\{{b_i(T), v(T) - v(T \setminus \{i\})\}}$ and also add brackets for the other \min commands in the doc